### PR TITLE
Added overwrite option to ExplicitSubscriber

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR introduces overwrite option to enhance the developer to overwrite a already existing node.

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@danrot can you add your thoughts about this change?

File: lib/Subscriber/Behavior/Path/ExplicitSubscriber.php [hunk 1]
```diff
@@ -64,12 +65,14 @@ public function configureOptions(ConfigureOptionsEvent $event)
             'node_name' => null,
             'parent_path' => null,
             'auto_create' => false,
+            'overwrite' => false,
```
Comment (danrot) [inline lib/Subscriber/Behavior/Path/ExplicitSubscriber.php]: Use `override`

**Review by danrot — CHANGES_REQUESTED** [review]

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by danrot** [comment]
Changelog missing

**Comment by wachterjohannes** [comment]
@danrot fixed

**Review by danrot — APPROVED** [review]

**Pull request merged**